gnomad_qc/v4/annotations/generate_freq_genomes.py

```diff
@@ -1459,6 +1550,15 @@ def main(args):
         ),
         action="store_true",
     )
+    parser.add_argument(
+        "--pop-diff-an-only",
+        help=(
+            "Only run --compute-allele-number-for-new-variants on the samples that "
```

Comment:
Can we make a `--compute-allele-number-for-pop-diff` separately? It seems less confusing.

Reply:
Or maybe change the args `--compute-allele-number-for-new-variants` to `--compute-allele-number-for-subsets`? It means subsets of samples and variants.

Reply:
I was trying not to copy and paste too much, and we are running low on time so this was a fast way to do what is needed. Please add as a suggestion or put a PR on top of this one to save me time